tombo/tombo_text_output.xml

```diff
@@ -52,12 +52,15 @@ ls -l ./
 
 &&
 
-cat *.wig > combined.wig
+#if $mode.file_types == 'coverage':
+    cat *.bedgraph > combined.bedwig
```

Comment:
bedwig is not a fileformat

And here is a difference between wig and bedgraph: https://www.biostars.org/p/16261/